src/components/HouseSlider.jsx

```diff
@@ -0,0 +1,78 @@
+import { useSelector, useDispatch } from 'react-redux';
+import { useEffect, useState } from 'react';
+import { NavLink } from 'react-router-dom';
+import { BiLeftArrow, BiRightArrow } from 'react-icons/bi';
+import { fetchHouses, fetchHouse } from '../redux/houses/housesSlice';
+import House from './House';
+
+const HouseSlider = () => {
+  const dispatch = useDispatch();
+  const { housesList } = useSelector((store) => store.houses);
+
+  const [indexOne, setIndexOne] = useState(0);
+  const [indexTwo, setIndexTwo] = useState(1);
+  const [indexThree, setIndexThree] = useState(2);
```

Comment:
Nice implementation on this useState hooks, It will be better to find a way to avoid hard coding everything. [This will help](https://stackoverflow.com/questions/59813926/how-can-i-store-and-update-multiple-values-in-react-usestate)

Reply:
Thank you! @SimonGideon, I will refactor these 3 states in the next branch following the information you shared.